ONSdigital/takeon-ui
# Feature/spp227/display se flag and score

# Description
 Added SE flag and score to the UI
 https://collaborate2.ons.gov.uk/jira/browse/SPP-227


# How to test?
 Visual check on the UI, new information displayed at the bottom of the contributor info panel, will be blank if no validation run, if validation and SE has run then it will display pass fail or missing and a numeric score
 https://dc-ui-spp227.es-take-on.aws.onsdigital.uk/



# Links
